spec/features/users/user_creates_new_account_spec.rb

```diff
@@ -16,7 +16,7 @@
     expect(page).not_to have_text(I18n.t(
       "flash.actions.create.notice",
       resource_name: "User",
-    ))
+    ),)
```

Comment:
And ^

Reply:
Done in PR ad-refactor-flash-format-specs sha: 4be77c8